File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int King = sc.nextInt();
        int Queen = sc.nextInt();
        int Rook = sc.nextInt();
        int Bishop = sc.nextInt();
        int Knight = sc.nextInt();
        int Pawn = sc.nextInt();
        
        System.out.println(1 - King);
        System.out.println(1 - Queen);
        System.out.println(2 - Rook);
        System.out.println(2 - Bishop);
        System.out.println(2 - Knight);
        System.out.println(8 - Pawn);
    }
}

File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.cs
using System;

class Q3003
{
    static void Main(string[] args)
    {
        string[] S = Console.ReadLine().Split();
        int King = int.Parse(S[0]);
        int Queen = int.Parse(S[1]);
        int Rook = int.Parse(S[2]);
        int Bishop = int.Parse(S[3]);
        int Knight = int.Parse(S[4]);
        int Pawn = int.Parse(S[5]);
        
        Console.Write((1 - King) + " ");
        Console.Write((1 - Queen) + " ");
        Console.Write((2 - Rook) + " ");
        Console.Write((2 - Bishop) + " ");
        Console.Write((2 - Knight) + " ");
        Console.Write(8 - Pawn);
    }
}
